scripts/Packages/Package_DFN_QFN/size_definitions/dfn.yaml

```diff
@@ -253,6 +253,46 @@ DFN-8-1EP_3x3mm_P0.65mm_EP1.7x2.05mm:
   #include_suffix_in_3dpath: 'False'
   #include_pad_size: 'fp_name_only' # 'both' | 'none' (default)
 
+DFN-10-1EP_2.5x2.5mm_P0.5mm_EP1.2x2.0mm:
```

Comment:
If we choose to use the texas naming convention for the footprint we might as well call the size definition entry by the same name.

```yaml
Texas_S-PDSO-N10_EP1.2x2.0mm
```